modules/odr_frontend/src/routes/dco/+page.svelte

```diff
@@ -0,0 +1,77 @@
+<script lang="ts">
+    import { enhance } from '$app/forms';
+
+    let acceptedDCO = false;
+</script>
+
+<svelte:head>
+    <title>DCO | OMI Data Pipeline</title>
+</svelte:head>
+
+<div class="flex flex-col overflow-y-auto">
+    <main class="container mx-auto p-4">
+        <h1 class="text-3xl font-bold mb-4">Developer Certificate of Origin</h1>
+        <h2 class="text-xl mb-2">Version 1.1</h2>
+
+        <div class="bg-gray-800 p-4 rounded-lg mb-4 whitespace-pre-wrap text-gray-200">
+{`Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
+1 Letterman Drive
+Suite D4700
+San Francisco, CA, 94129
+
+Everyone is permitted to copy and distribute verbatim copies of this
+license document, but changing it is not allowed.
+
+
+Developer's Certificate of Origin 1.1
+
```

Comment:
Do we cite the "open source license indicated in the file" anywhere in this screen? We might want to clarify that point. If this will be a dataset, I believe its CDLA 2.0

Reply:
@hipsterusername No we don't mention the specific license anywhere. 

To clarify are you suggesting updating the wording of the DCO here to add a line specifying the license, or should we just have a line above/before the DCO block stating like "The following Developer Certificate of Origin agreement covers all contributions to Open Model Initiative Datasets. All OMI datasets will fall under the CDLA 2.0 license" perhaps with a link to https://cdla.dev/permissive-2-0/ ?

Just wanting to make sure I understand the spec or expected wording updates correctly.

Reply:
Yes I think we just need to highlight which license applies in the UI somewhere

Reply:
Okay thanks, I will add the line and link in a quick subsequent PR. 👍